release/start-secured-cluster/patch-monitoring.json

```diff
@@ -0,0 +1,23 @@
+{
+    "spec": {
+        "template": {
+            "spec": {
+                "containers": [
+                    {
+                        "name": "prometheus",
+                        "resources": {
+                            "requests": {
+                                "memory": "2Gi",
```

Comment:
How about we request 8Gi to prevent OOMKills because the node runs out of memory, if it is overcommitted? 

If you change this here, please verify in another long-running cluster that Prometheus is starting (as I am unsure if there are 8Gi available on each node with ACS' requirements too).